scripts/prepare_gas_network.py

```diff
@@ -458,12 +458,17 @@ def get_GADM_layer(
             output_nonstd_to_csv=False,
         )
 
-        # create a subindex column that is useful
-        # in the GADM processing of sub-national zones
-        # Fix issues with missing "." in selected cases
-        geodf_temp["GADM_ID"] = geodf_temp[f"GID_{cur_layer_id}"].apply(
-            lambda x: x if x[3] == "." else x[:3] + "." + x[3:]
-        )
+        if layer_id == 0:
+            geodf_temp["GADM_ID"] = geodf_temp[f"GID_{cur_layer_id}"].apply(
+                lambda x: x + ".1"
+            )
```

Comment:
Since we have a single country here, if there are lines they should be duplicates.
Maybe something like the following should work:
```suggestion
            geodf_temp["GADM_ID"] = (
                geodf_temp[f"GID_{cur_layer_id}"] +
                pd.Series(range(1, geodf_temp.shape[0]+1)).astype(str)
            )
```
Could you check?